src/data/full-quiz.js

```diff
@@ -1,4 +1,40 @@
 const fullQuiz = [
+  {
+    Question: "What is a maximum length for a filename under Linux?",
+    Answer: "255 Bytes",
+    Distractor1: "225 Bytes",
+    Distractor2: "235 Bytes",
+    Distractor3: "295 Bytes",
+    Explanation: "The maximum length for a file name is 255 bytes. The maximum combined length of both the file name and path name is 4096 bytes.",
+    Link: "https://www.ibm.com/docs/en/spectrum-protect/8.1.9?topic=parameters-file-specification-syntax",
+  },
+  {
+    Question: "An Example for LIFO structure is",
+    Answer: "Stacks",
+    Distractor1: "Priority Queues",
+    Distractor2: "Trees",
+    Distractor3: "Queues",
+    Explanation: "Stack is a linear data structure which follows a particular order in which the operations are performed. The order may be LIFO(last in first out) or FILO(First In Last Out)",
+    Link: "https://www.geeksforgeeks.org/stack-data-structure/",
+  },
+  {
+    Question: "Which of the following is a Divide and Conquer algorithm?",
+    Answer: "Merge Sort",
+    Distractor1: "Selection Sort",
+    Distractor2: "Heap Sort",
+    Distractor3: "Bubble Sort",
+    Explanation: "Merge Sort is a Divide and Conquer algorithm. It divides the input array into two halves, calls itself for the two halves, and then merges the two sorted halves.",
+    Link: "https://www.geeksforgeeks.org/merge-sort/",
+  },
+  {
+    Question: "Worst case time complexity to access an element in a BST can be?",

```

Comment:
Proposed fix:

What is the worst case time complexity for accessing an element in a BST(Binary Search Tree)?